resources/sass/_variables.scss

```diff
@@ -1,17 +1,17 @@
 // Colors
-$blue: #3490dc;
-$indigo: #6574cd;
-$purple: #9561e2;
-$pink: #f66D9b;
-$red: #e3342f;
-$orange: #f6993f;
-$yellow: #ffed4a;
-$green: #38c172;
-$teal: #4dc0b5;
-$cyan: #6cb2eb;
+// $blue:
+// $indigo:
+// $purple:
+// $pink:
+// $red:
+// $orange:
+// $yellow:
+// $green:
+// $teal:
+// $cyan:
```

Comment:
Shouldn't we define the used colors as variables here, in replacement of these ones, so future use are consistent?

Reply:
Would we be able to do that _and_ integrate the dark-mode variables [as suggested by @vergissberling](https://github.com/monicahq/monica/issues/1192#issuecomment-754507231)?  I'd actually planned to move over to using `--var`-style names in the dark-mode conversion pull using that technique.

I don't really understand SASS, so it's possible this can be done.  I just don't know enough to say either way.

Reply:
Actually, CSS custom properties (or variables as they are commonly called) are available in vanilla CSS. No preprocessor like SASS needed. 

With the notable exception of Internet Explorer which lacks support, everything else is compatible. 

And, yes, they would be quite helpful for an easier dark-mode implementation. Good inspiration from [this article](https://mwichary.medium.com/dark-theme-in-a-day-3518dde2955a).

Reply:
I much prefer CSS variables to SASS's equivalent -- if only because I understand them and have experience of them.  An earlier comment [by @vergissberlin](https://github.com/monicahq/monica/issues/1192#issuecomment-754507231) shows how to use them really efficiently for dark mode.

Naming them may be a little tricky, since the colours switch between light and dark modes, but it's not impossible.

Once we have the simplified colours in master, I’ll start a new branch to do dark-mode.  My grasp of git is too wobbly to try and do it on a fork that isn't forked from master.

Reply:
OK let's merge this one